### PR TITLE
open-cluster-management/backlog#1861 Add ts unknown

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: troubleshoot_acm/troubleshooting.md [hunk 1]
```diff
@@ -149,7 +151,63 @@ After importing a cluster by using the Red Hat Advanced Cluster Management conso
...
+
+  1. Run the following command on the hub cluster:
+  
+    ```
+    oc describe clusterdeployments -n <new_cluster_name>
+    ```
+  
+    Replace *new_cluster_name* with the name of the cluster that you created.
+	
+  2. See if there is additional information about the problem in the _Status.Conditions.Message_ and _Status.Conditions.Reason_ entries of the logs.
```
Comment (chrisahl) [inline troubleshoot_acm/troubleshooting.md:196]: The user will be looking at the ClusterDeployment resource, not logs so,
Change from:
entires of the logs.
to:
entires of the resource

File: troubleshoot_acm/troubleshooting.md [hunk 2]
```diff
@@ -149,7 +151,63 @@ After importing a cluster by using the Red Hat Advanced Cluster Management conso
...
+
+#### Symptom
+
+After creating a new cluster by using the Red Hat Advanced Cluster Management for Kubernetes console, the cluster shows a status of _Unknown_ in the console.
+
+#### Identifying the problem
+
+* Procedure 1
+
+  1. Run the following command on the hub cluster to view the names of the Kubernetes pods that have the status of _Unknown_:
```
Comment (chrisahl) [inline troubleshoot_acm/troubleshooting.md:166]: The _Unknown_ status will not show for the pod(s).  Propose change is 
from:
Kubernetes pods that have the status of _Unknown_
to:
Kubernetes pods that have been created in the new cluster's namespace:

File: troubleshoot_acm/troubleshooting.md (hunk 1 above)
Comment (cadawson) [reply troubleshoot_acm/troubleshooting.md]: Fixed

File: troubleshoot_acm/troubleshooting.md (hunk 2 above)
Comment (cadawson) [reply troubleshoot_acm/troubleshooting.md]: Updated

**Pull request merged**